src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -868,26 +905,41 @@ type TogglerTypes = OmitHtmlDivProps<{
 
 const SpanTreeTogglerContainer = styled('div')<TogglerTypes>`
   position: relative;
-  height: 16px;
+  height: ${SPAN_ROW_HEIGHT}px;
   width: ${p => (p.hasToggler ? '40px' : '12px')};
   min-width: ${p => (p.hasToggler ? '40px' : '12px')};
   margin-right: ${p => (p.hasToggler ? space(0.5) : space(1))};
   z-index: ${zIndex.spanTreeToggler};
   display: flex;
   justify-content: flex-end;
+  align-items: center;
 `;
 
-const SpanTreeConnector = styled('div')<TogglerTypes>`
-  height: ${p => (p.isLast ? '80%' : '160%')};
+const SpanTreeConnector = styled('div')<TogglerTypes & {orphanBranch: boolean}>`
+  height: ${p => (p.isLast ? SPAN_ROW_HEIGHT / 2 : SPAN_ROW_HEIGHT)}px;
   width: 100%;
-  border-left: 1px solid ${p => p.theme.gray1};
+  border-left: ${p => {
+    if (p.orphanBranch) {
+      return `1px dashed ${p.theme.gray1}`;
+    }
+
+    return `1px solid ${p.theme.gray1}`;
+  }};
+
```

Comment:
```suggestion
  border-left: 1px ${p => p.orphanBranch ? 'dashed' : 'solid'} ${p => p.theme.gray1};
```
A ternary saves you some lines and duplication.